tests/bats/Makefile

```diff
@@ -62,7 +62,7 @@ tests: image
 		$(BATS_IMAGE) \
 		-c "cd /cwd; \
 			echo 'Running k8s cluster cleanup (invasive)... '; \
-			bash tests/bats/cleanup-from-previous-run.sh &> $${_RUNDIR}/cleanup.outerr || \
+			bash tests/bats/cleanup-from-previous-run.sh 2>&1 | tee -a $${_RUNDIR}/cleanup.outerr || \
```

Comment:
Before: show output on terminal on non-zero exit code.
Now: always show output.

For the happy path, I reduced the noisiness of that script. 

Revising an intuition from a couple days ago: I now want to see that output on the terminal every time. To not get too detached from how well (or extremely unwell) this cleanup is actually doing.